spec/controllers/file_controller_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'file_controller'
 
 describe FileController do
```

Comment:
I think you might be able to ditch the require statement above if you use:
`RSpec.describe FileController, type: :controller do` instead